web-components/navdots/navdots.stories.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+  import NavDots from './navdots.svelte'
+
+  let activeDot = 0
+  function handleChange(n) {
+    activeDot = n.detail.activeDot
+  }
+</script>
+
+<Meta
+  title="Nav Dots"
+  component={NavDots}
+  argTypes={{
+    dots: { control: 'number', defaultValue: 10 },
+    dotSize: { control: 'number', defaultValue: 8 },
+    activeColor: { control: 'color' },
+    defaultColor: { control: 'color' },
+  }}
+/>
+
+<Template let:args>
+  <div style={`--color-interaction-button-primary-background: ${args.activeColor || 'unset'}; --color-primary-20: ${args.defaultColor || 'unset'}`}>
```

Comment:
What do you think about making a component-specific variable that can be overriden, e.g. `--color-navdots`. I worry about the parents knowing the specific variable that gets used.

Reply:
Separately, and this is also a question for @aguscruiz - do we need to allow overriding _active_ dot color as well as dot color, or is there some kind of consistent calculation from dot-color to active-dot-color, like opacity?

Reply:
Yeah, it'd be really neat if everything derived from one color :smile: